src/core/python2js.c

```diff
@@ -486,6 +492,9 @@ _python2js(ConversionContext *context, PyObject* x)
   } else {
     context->depth--;
     JsVal result = _python2js_deep(context, x);
+    if (context->proxies && pyproxy_Check(result)) {
+      JsvArray_Push(hiwire_get(context->proxies), result);
+    }
```

Comment:
What's the impact if this is missed? Curious because it seems this is no longer done after every call to `_python2js_deep` (should it be done in `python2js_custom__create_jscontext` too?)

Reply:
Well it's for management of the proxy lifetime -- the list of proxies can be destroyed all at once. The leak checks in the test suite should automatically catch bugs here, so hopefully that means this isn't wrong.

> this is no longer done after every call to `_python2js_deep`

I'm not sure what you mean by this?

> should it be done in python2js_custom__create_jscontext too?

This list is for PyProxies that are created and handed over to the caller for them to manage. The `jscontext` is an implementation detail of the conversion itself and the `python2js` code destroys it manually when done converting.

Reply:
> I'm not sure what you mean by this?

I just meant I didn't see the proxy getting added to the context in `python2js_custom__create_jscontext`. But what you say makes sense.